### ISSUE TITLE
UI: Update Admin Tools Scan History Table

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#343]
# 🐛 Summary #

The Admin Tools  Scan History table needs to be updated to the MUI Data Grid component to stay inline with the rest of the app.

It will also need proper aria-labels and tab indexes for assistive technology users to meet 508 compliance:

- “Details Column “”” buttons inside the Scan History table are not in keyboard tab order. There is no mechanism in which assistive technology users can select the “” button because the interactive element is not keyboard accessible using the either the tab keys or the arrow keys. This happens because the <svg> is coded within a non-interactive <span> element rather than as a <button>.
Add any screenshots of the problem here.

- The “Details” column “” buttons inside the Scan history table do not have accessible name specified for the buttons. Each “” button should have a Unique identifier to distinguish it from the other rows in the table. For example: the aria-label for the first row should be “c488cc67-48d4-4c67-b40b-45d7d5e028a5 Toggle Row Collapsed” (assuming that ID is a unique value.)

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/7f3f32e0-3a25-4739-a170-9768465714ce)
